docs/devices/TH1300ZB.md

```diff
@@ -12,12 +12,11 @@ description: "Integrate your Sinope TH1300ZB via Zigbee2MQTT with whatever smart
 | Model | TH1300ZB  |
 | Vendor  | Sinope  |
 | Description | Zigbee smart floor heating thermostat |
-| Exposes | local_temperature, lock (state), climate (occupied_heating_setpoint, local_temperature, system_mode, running_state, pi_heating_demand), backlight_auto_dim, linkquality |
```

Comment:
Why was this changed?

Reply:
I don't know, Why other files changed...
I only add 2files(SG-V100-ZB.md, SG-V100-ZB.jpg)